Solutions/PaloAlto-PAN-OS/Workbooks/PaloAltoOverview.json

```diff
@@ -1503,7 +1203,9 @@
       "name": "Compare allowed and denied files by time"
     }
   ],
-  "styleSettings": {},
+  "fallbackResourceIds": [
+    "/subscriptions/4383ac89-7cd1-48c1-8061-b0b3c5ccfd97/resourcegroups/v-muuppugund/providers/microsoft.operationalinsights/workspaces/testsentinel"
```

Comment:
Check on the subscription and resource id, it should not be hardcoded here.

Reply:
By mistake it has been checked in already corrected